main/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/stackrox/CvesPusher/utils"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+)
+
+const (
+	malformedCVEFile = "scratch/malformedCVEs"
+	correctCVEFile   = "scratch/correctCVEs"
+)
+
+func TestEnvVarErrors(t *testing.T) {
+	_, err := utils.IsEnvVarNonEmpty("SOME_LOCAL_PATH")
```

Comment:
If you're going to test env vars, you need to set them first. (I shouldn't have to export some var before I run the tests)